siriuspy/siriuspy/orbintlk/main.py

```diff
@@ -254,7 +254,7 @@ def fambpm_dev(self):
     def set_enable(self, value):
         """Set orbit interlock state.
         Configure global BPM interlock enable and EVG interlock enable."""
-        if value not in _ETypes.DSBL_ENBL:
+        if not 0 <= value < len(_ETypes.DSBL_ENBL):
```

Comment:
why was this a bug?

Reply:
because the value is an index of the tuple and the _ETypes.DSBL_ENBL is a tuple of strings

Reply:
right!